homeassistant/components/binary_sensor/modbus.py

```diff
@@ -16,11 +16,13 @@
 _LOGGER = logging.getLogger(__name__)
 DEPENDENCIES = ['modbus']
 
+CONF_HUB = 'hub'
 CONF_COIL = 'coil'
 CONF_COILS = 'coils'
 
 PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
     vol.Required(CONF_COILS): [{
+        vol.Required(CONF_HUB, default='default'): cv.string,
```

Comment:
The default value should also be defined centrally.

Reply:
Done. Defined DEFAULT_HUB.